openquake/hazardlib/gsim/mgmpe/nrcan15_site_term.py

```diff
@@ -0,0 +1,167 @@
+# The Hazard Library
+# Copyright (C) 2012-2018 GEM Foundation
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module :mod:`openquake.hazardlib.mgmp.nrcan15_site_term` implements
+:class:`~openquake.hazardlib.mgmpe.NRCan15SiteTerm`
+"""
+
+import copy
+import numpy as np
+from openquake.hazardlib.gsim.base import CoeffsTable
+from openquake.hazardlib.gsim.base import GMPE, registry
+from openquake.hazardlib.gsim.boore_atkinson_2008 import BooreAtkinson2008
+
+
+class NRCan15SiteTerm(GMPE):
+    """
+    Implements a modified GMPE class that can be used to account for local
+    soil conditions in the estimation of ground motion.
+
+    :param gmpe:
+        An instance of the :class:`~openquake.hazardlib.gsim.base.GMPE`
```

Comment:
I would change the parameter `gmpe` -> `gmpe_name` since it is not an instance of GMPE.